hw/top_earlgrey/dv/vendor_chip_sim_cfg_sample.hjson

```diff
@@ -0,0 +1,15 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  // Import additional common sim cfg files.
+  import_cfgs: ["{proj_root}/hw/top_earlgrey/dv/chip_sim_cfg.hjson"]
+  overrides: [
+    {
+      name: fusesoc_cores_root_dirs
```

Comment:
The overrides directive allows you to modify an existing key (that defaults to something) with (an) alternate value(s).

Reply:
awesome, thanks @sriyerg

Reply:
just to confirm, they could also override the fusesoc command line options this way is that right?

Reply:
Any sim cfg hjson variable can be overridden. The way I see it, overriding either the fusesoc core or the search paths should be sufficient, agree?

Reply:
Yep! definitely works! Just wanted to make sure as I write-up the instructions for Nuvoton. (since I point to this PR as an example).

Reply:
Oh, but you are also right - they can completely change the fusesoc command line if they wanted to (by overriding the rest of the `sv_flist_gen_*` variables). In fact they can even replace fusesoc with some other dependency resolver system such as Bazel if they wanted to.